ui/raidboss/data/03-hw/raid/a1s.js

```diff
@@ -89,7 +90,7 @@
       id: 'A1S Hypercompressed Collect',
       regex: Regexes.startsUsing({ id: 'E4A', source: ['Oppressor', 'Oppressor 0\.5'] }),
       regexDe: Regexes.startsUsing({ id: 'E4A', source: ['Unterdrücker', 'Unterdrücker 0,5'] }),
-      regexFr: Regexes.startsUsing({ id: 'E4A', source: ['Oppresseur', 'Oppresseur 0\\.5'] }),
+      regexFr: Regexes.startsUsing({ id: 'E4A', source: ['Oppresseur', 'Oppresseur 0\.5'] }),
```

Comment:
I think everything should have two backslashes rather than taking one away from the French regex.